explore/src/main/scala/explore/TopBar.scala

```diff
@@ -94,24 +95,27 @@ object TopBar {
                            simple = true,
                            compact = true,
                            icon = Icons.Bars,
+                           open = false,
                            clazz = ExploreStyles.MainMenuDropdown
                   )(
                     DropdownMenu(
                       About(
-                        DropdownItem(text = "About Explore", icon = Icons.Info),
-                        <.span(ExploreStyles.Version,
-                               ExploreStyles.VersionUncopied.when(! $.state.copied)
-                        )(
-                          s"Version: ${appCtx.version}",
-                          CopyToClipboard(
-                            text = appCtx.version.value,
-                            onCopy = (_, copied) =>
-                              $.setStateL(State.copied)(copied) >>
-                                $.setStateL(State.copied)(false).delayMs(1500).toCallback
+                        Reuse.always(DropdownItem(text = "About Explore", icon = Icons.Info)),
```

Comment:
Just FYI: You can also do `whatever.reuseAlways`.

Reply:
Nice